server/bin/docker/genCerts.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+WORKING="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. ${WORKING}/deploy-env.sh
+
+PASSPREFIX=pass:
+KEYPASS=password
+PASSWORD=$PASSPREFIX$KEYPASS
+
+if [ ! -d ${SRC_DIR}/pki ]; then
+	mkdir ${SRC_DIR}/pki
+	pushd ${SRC_DIR}/pki
+
+	#Create a private key
+
+	openssl genrsa -des3 -out CA.key -passout ${PASSWORD} 4096
```

Comment:
Switched to -aes256.  Was just following the steps in the [timely](https://nationalsecurityagency.github.io/timely/docs/#ssl-setup) doc, which was using the steps in the [accumulo](http://accumulo.apache.org/1.8/accumulo_user_manual#_generating_ssl_material_using_openssl) doc.